### PR TITLE
dashboardframe refactored to import components

### PR DESCRIPTION
**Pull request by VeraNemedyVarga**
..plus discussion element template created, dashboardframe.js renders it in for loop for an arbitrary number.

### CODE REVIEW

**Comment by ilcsikbalazs** [comment]
conflicts resolved

File: src/js/dashboardframe_sidebar.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    ProfileOnClick() {
+        console.log("ProfileOnClick");
+    }
+
+    AllDiscussionsOnClick() {
+        console.log("AllDiscussionsOnClick");
+    }
+
+    MyDiscussionsOnClick() {
```
Comment (tiborsaas) [inline src/js/dashboardframe_sidebar.js]: Please follow style guide: methods are not capitalized

File: src/js/login.js [hunk 2]
```diff
@@ -7,10 +7,15 @@ class Login extends Component {
         this.state = {
             inputLoginValue: '',
             inputPassValue: '',
+<<<<<<< HEAD
```
Comment (tiborsaas) [inline src/js/login.js]: meh, resolve merge conflicts PLZZZ

File: src/js/login.js [hunk 3]
```diff
@@ -76,7 +67,7 @@ class Login extends Component {
                         <input className="login_pass" type="password" placeholder="password" value={this.state.inputPassValue} onChange={this.handleOnChangePass.bind(this)}></input>
                         <div className="container__button">
                             <button className="login_button"  onClick={this.LoginOnClick.bind(this)}>LOGIN</button>
-                            <button className="register_button" value={this.state.registerClickHappened} onClick={this.RegisterOnClick.bind(this)}>REGISTER</button>
+                            <button className="register_button" onClick={this.RegisterOnClick.bind(this)}>REGISTER</button>
```
Comment (tiborsaas) [inline src/js/login.js]: you forget to make the "RegisterOnClick" lowercase.

**Pull request merged**